### ISSUE TITLE
Wrong validation rule linked

### ISSUE THREAD

**Issue by smares** [laravel-idea/plugin#346]
Correct:

![image](https://user-images.githubusercontent.com/818380/126790187-cd310eef-66b3-4a47-9a95-52e2acaf46a8.png)

Wrong:

![image](https://user-images.githubusercontent.com/818380/126790260-13affeef-192d-4379-86ac-dce800171437.png)

![image](https://user-images.githubusercontent.com/818380/126790041-042ec1be-63b7-4321-b9fa-ed0bafd77e7b.png)

Happens with 4.3.4.211

**Comment by adelf** [comment]
I can't reproduce it. Weird issue...
Honestly, I can't imagine how this code can find another rule:
```kotlin
fun Project.getReferenceForRule(name: String): PsiElement? {
    val standardMethod = getStandardValidationRules(this).filter {
        it.name == name
    }.map { it.method }.firstOrNull()

    if (standardMethod != null) return standardMethod
```
getStandardValidationRules() returns the list of objects:
```
{name: 'required', method: PHP method(validateRequired)}
{name: 'required_if', method: PHP method(validateRequiredIf)}
```
And the name is calculated from PHP method name: substring(8) and toSnakeCase()...

I can't even suggest anything...

**Comment by smares** [comment]
I am managing the `vendor` directory via `composer` on the remote machine where I sync my code changes to via SFTP.

Every now and then I have to tell PhpStorm to download the remote `vendor` directory to my local code base so that I can use code completion locally. I did this earlier today but didn't close PhpStorm since then.

So maybe some kind of caching issue? I noticed that the file with the validation rules did change today.

**Comment by adelf** [comment]
> So maybe some kind of caching issue?

Yes. It also might be the reason for your multiple view issue... I'll try to find and fix it.

Also, I'll add a special "Invalidate Laravel Idea caches" command, which also will help in some cases.

**Comment by gorbunov** [comment]
Cache clearing command should resolve similar issues
